### PR TITLE
fix: revive the ability of a zoe client to get access to the code.

### LINKED ISSUE #877: How can i get the contract source code from an invite?
- From the `invite` payment amount, i can extract the `instanceHandle`
- I can call `zoe.getInstance(instanceHandle)` which gives me the `installationHandle`

But from the `installationHandle`, it's not clear to me how a user can get the contract source code

In any case, going through 2 handles to get to the source code feels like too much for no good reason

Could the source code be made available directly as part of the instanceRecord?

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
It's tested in an existing large test by checking for specific text in
the source code. We could check something less fragile (e.g. starts
with 'function getExport'), and still be pretty sure that if a string
is returned, it would continue to be the source of the contract.

fixes #877

### CODE REVIEW

File: packages/zoe/src/zoe.js [hunk 1]
```diff
@@ -321,7 +321,7 @@ const makeZoe = (additionalEndowments = {}) => {
         }
       }
       const installationHandle = installationTable.create(
-        harden({ installation }),
+        harden({ installation, code }),
```
Comment (katelynsills) [inline packages/zoe/src/zoe.js:324]: What's the difference between the installation and the code?

File: packages/zoe/src/state.js [hunk 2]
```diff
@@ -8,7 +8,9 @@ import { makeTable, makeValidateProperties } from './table';
 // Installation Table
 // Columns: handle | installation
```
Comment (erights) [inline packages/zoe/src/state.js]: Looks like you need to add `code` to this comment as an additional column.

**Review by erights — DISMISSED** [review]

File: packages/zoe/src/zoe.js (hunk 1 above)
Comment (Chris-Hibbert) [reply packages/zoe/src/zoe.js:324]: `code` is a string with the actual source code. `installation` is the result of evaluating the code with all the context it needs. It's an object with a `makeContract()` function. That function has a bunch of metering wrappers thrown in, so it's not a useful place for clients to get access to and compare the code they are expecting to see.

File: packages/zoe/src/state.js (hunk 2 above)
Comment (Chris-Hibbert) [reply packages/zoe/src/state.js]: done

**Review by katelynsills — APPROVED** [review]
LGTM!

**Pull request merged**